app/components/explore/side-bar.js

```diff
@@ -172,6 +182,8 @@ export default class SideBar extends Component {
       cfs               : null,
       event_name        : null,
       is_online         : null,
+      has_logo          : null,
+      has_image         : null,
```

Comment:
Still wrong indentation

Reply:
Sir, i am not able to figure out what is wrong with identation. Could you please give hint or suggestion